gr-trt.lwr

```diff
@@ -0,0 +1,29 @@
+#
+# This file is part of PyBOMBS
+#
+# PyBOMBS is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# PyBOMBS is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PyBOMBS; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+#
+
+category: common
+depends:
+# Basic Dependencies
+- gnuradio
+- TensorRT
```

Comment:
Is there a recipe for TensorRT somewhere?

Reply:
Nope. I just want my oot to show up on cgran.

Reply:
And there lies the issue, I'm hoping to start on CGRANv2 soon which will decouple it from PyBOMBS, but for now, merging!